From foo@baz Mon Aug 10 10:46:37 PDT 2009
Date: Mon, 10 Aug 2009 10:46:37 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: echo: remove TODO item

The proc interface is no longer in the echo code, so remove the TODO
item.

Cc: Steve Underwood <steveu@coppice.org>
Cc: David Rowe <david@rowetel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/echo/TODO |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/echo/TODO
+++ b/drivers/staging/echo/TODO
@@ -1,7 +1,5 @@
 TODO:
 	- handle bit_operations.h (merge in or make part of common code?)
-	- remove proc interface, only use echo.h interface (proc interface is
-	  racy and not correct.)
 
 Please send patches to Greg Kroah-Hartman <greg@kroah.com> and Cc: Steve
 Underwood <steveu@coppice.org> and David Rowe <david@rowetel.com>
